packages/manager/apps/hub/package.json

```diff
@@ -11,9 +11,10 @@
   "license": "BSD-3-Clause",
   "author": "OVH SAS",
   "scripts": {
-    "build": "webpack --env.production",
+    "build": "webpack --env.production && yarn run generate:manifest",
```

Comment:
No need to generate in  dev mode ?

Reply:
As it parse the `dist` it won't work in dev.